readme.md

```diff
@@ -104,19 +110,30 @@ Default: `false`
 
 Use full-length units: `5h 1m 45s` → `5 hours 1 minute 45 seconds`
 
+Setting `colonNotation` to `true` overrides this option.
+
 ##### separateMilliseconds
 
 Type: `boolean`\
 Default: `false`
 
 Show milliseconds separately. This means they won't be included in the decimal part of the seconds.
 
+Setting `colonNotation` to `true` overrides this option.
```

Comment:
Instead of writing this for each option, I think it would be better to just document in the `colonNotation` docs what options it overrides.